model/src/main/java/de/danoeh/antennapod/model/feed/FeedPreferences.java

```diff
@@ -78,10 +78,33 @@ public static SkipSilence fromCode(int code) {
         }
     }
 
+    public enum AutoDownloadSetting {
+        DISABLED(0),
+        ENABLED(2),
+        // the reason for this is that the database column uses 1 as the default value.
+        // migrating the cells to "2" is more easy than changing the default value.
```

Comment:
This comment is useful for reviewing, but does not need to be kept in the source code forever. Please remove for the final version